ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -1257,3 +1257,48 @@ def test_torch_multi_dot(
         test_values=True,
         tensors=x,
     )
+
+# solve_ex
+@handle_frontend_test(
+    fn_tree="torch.linalg.solve_ex",
+    dtype_and_data=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=10,
+        shape=helpers.ints(min_value=2, max_value=5).map(lambda x: tuple([x, x + 1])),
+        safety_factor_scale="log",
+        small_abs_safety_factor=6,
+    ).filter(
+        lambda x: "float16" not in x[0]
+        and "bfloat16" not in x[0]
+        and np.linalg.cond(x[1][0][:, :-1]) < 1 / sys.float_info.epsilon
+        and np.linalg.det(x[1][0][:, :-1]) != 0
+        and np.linalg.cond(x[1][0][:, -1].reshape(-1, 1)) < 1 / sys.float_info.epsilon
+    ),
+    dtype_and_data2=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("bool")),
+)
+def test_torch_solve_ex(
+    *,
+    dtype_and_data,
+    dtype_and_data2,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+    input_dtype, data = dtype_and_data
+    input = data[0][:, :-1]
+    other = data[0][:, -1].reshape(-1, 1)
+    check_dtype, check = dtype_and_data2
+    check = check[0]
+    test_flags.num_positional_args = 3
```

Comment:
There shouldn't be a need to add this as the `test_function` is able to figure out the number of positional/keyword-only arguments to a function.